Osiris/Config.cpp

```diff
@@ -1104,6 +1105,45 @@ void Config::save(size_t id) const noexcept
     }
 }
 
+bool Config::toClipboard() const noexcept
+{
+    std::stringstream ss;
+
+    json j;
+
+    j["Aimbot"] = aimbot;
+    j["Triggerbot"] = triggerbot;
+    j["Backtrack"] = backtrack;
+    j["Anti aim"] = antiAim;
+    j["Glow"] = glow;
+    j["Chams"] = chams;
+    j["ESP"] = streamProofESP;
+    j["Sound"] = sound;
+    j["Visuals"] = visuals;
+    j["Misc"] = misc;
+    j["Style"] = style;
+    j["Skin changer"] = skinChanger;
+
+    removeEmptyObjects(j);
+    ss << std::setw(2) << j;
+
+    void* data = GlobalAlloc(GMEM_FIXED, sizeof(char) * ss.str().length() + 1);
```

Comment:
Documentation says that memory to be used in `SetClipboardData()` should be allocated with GMEM_MOVEABLE flag.